Containerfile.bpfman.local

```diff
@@ -24,7 +24,7 @@ RUN --mount=type=cache,target=/usr/src/bpfman/target/ \
     cp /usr/src/bpfman/target/release/bpfman-rpc ./bpfman/
 
 ## Image for Local testing is much more of a debug image, give it bpftool and tcpdump
-FROM fedora:41
+FROM fedora:42
```

Comment:
What are the benefits of using such a bleeding-edge version of Fedora? Does it provide newer toolchains or other advantages?

Reply:
Is 41 even officially released yet?  Also, why is Konflux doing this update?